html5/_main_toc.html.slim

```diff
@@ -35,7 +35,7 @@ nav#toc.collapse.bd-links.toc
       ul.nav.navbar-nav
         li.bd-sidenav-active
           a.nav-link href="https://docs.killbill.io/latest/userguide_subscription.html"
-            | Subscription
+            | Subscription 
```

Comment:
Somehow extra spaces were added (here and below), this might impact the HTML generation. Could you remove them?

Reply:
I've added these deliberately, if you look at the left nav, there is no space between the words **Subscription** and **Billing** which I thought was a typo. If not, please let me know and I will revert the changes.

Reply:
Oh, I see -- got it. Good point.

Unfortunately, these trailing spaces will likely be stripped out automatically by our IDEs. Can you specify `&nbsp;` instead (the HTML entity)?

Reply:
Fixed